data/profiles/default.yaml

```diff
@@ -292,6 +285,11 @@ spec:
               suffix: global
             # Domain name for telemetry addons
             telemetry_domain_name: ""
+            env:
+              # A gateway with this mode ensures that pilot generates an additional
```

Comment:
Let's keep comments out of these configs - it makes for extremely huge files and we shouldn't be mixing schema and data anyway (as values.yaml does).